### PR TITLE
Add Interface for InternalPrice Model

### PR DESCRIPTION
**Pull request by EUdds**
Saw there was an abandoned PR for this (#64 ), wanted this feature for an app I'm working on

### CODE REVIEW

File: inventree/part.py [hunk 1]
```diff
@@ -111,6 +111,20 @@ def uploadImage(self, image):
 
         return response
 
+    def getInternalPrice(self):
```
Comment (SchrodingersGat) [inline inventree/part.py:114]: I think this method would be better named either:

- `getInternalPrices`
- `getInternalPriceList`

Comment (EUdds) [reply inventree/part.py]: Noted

**Comment by EUdds** [comment]
Addressed your comment, lmk if there's anything else needed to kick off the workflow tests

**Comment by SchrodingersGat** [comment]
Still some unit tests failing here :)

**Comment by EUdds** [comment]
Doh - I'd love to pick your brain sometime about your dev environment, the tests are passing running on my local inventree server 🙃

**Comment by SchrodingersGat** [comment]
Interesting. Might be a slight difference in package setup? 

You can see the environment info here: https://github.com/inventree/inventree-python/blob/master/.github/workflows/ci.yaml

**Comment by SchrodingersGat** [comment]
Thanks for the contribution :)

**Pull request merged**